service/season_generator.py

```diff
@@ -40,6 +43,121 @@ async def check_season_end(self):
     async def generate_season(self):
         self._logger.info("Generating new season...")
         async with self._db.acquire() as conn:
-            with open("service/generate_season.sql") as file:
-                query = text(file.read())
-                await conn.execute(query)
+            sql = select([league]).where(league.c.enabled == True)
+            result = await conn.execute(sql)
+            rows = await result.fetchall()
+
+            now = datetime.now()
+            year = now.year
+            month = now.month
+            if month == 12:
+                month = 1
+                year += 1
+            else:
+                month += 1
```

Comment:
This code is just trying to add one month to the date right? How about just using a `relativedelta(months=1)`?
```suggestion
            now = datetime.now() + relativedelta(months=1)
            year = now.year
            month = now.month
```

Then you probably don't even need to assign `year` and `month` to variables

Reply:
Good idea. I think I did this before I discovered the relativedelta package.